Firestore/core/src/firebase/firestore/remote/grpc_root_certificate_finder_apple.mm

```diff
@@ -54,24 +54,22 @@
     if (!bundle) {
       continue;
     }
-
-    NSString* resource = @"gRPCCertificates.bundle/roots";
-    NSString* path = [bundle pathForResource:resource ofType:@"pem"];
-    if (!path) {
-      resource = @"gRPCCertificates-Firestore.bundle/roots";
-      path = [bundle pathForResource:resource ofType:@"pem"];
-    }
-
-    if (path) {
-      LOG_DEBUG("%s.pem found in bundle %s", resource,
-                [bundle bundleIdentifier]);
-      return path;
-    } else {
-      LOG_DEBUG("%s.pem not found in bundle %s", resource,
-                [bundle bundleIdentifier]);
+    // search for the roots.pem file in each of these resource locations
+    NSArray* possibleResources = @[@"gRPCCertificates.bundle/roots",
```

Comment:
nit: This could be hoisted out of the loop.

Reply:
Not if it's going to include the resource path in the message, which it was before